docs/changelog.md

```diff
@@ -43,6 +43,7 @@
 -   Added individual attribution to edits made on Note objects.
 -   Added the ICS Security Control mapping field to Mitigation objects.
 -   Updated the documentation for installing Workbench with additional certs.
+-   Added ability to search objects by the user that created them.
```

Comment:
Now that we've done a release, this line needs to be moved to a new `v1.3.2` section.